composer.json

```diff
@@ -13,9 +13,9 @@
     }
   ],
   "require": {
-    "drupal/core": "^8.8",
-    "drupal/components": "~1.0",
-    "drupal/ds": "~3.3",
-    "drupal/ui_patterns": "~1.0"
+    "drupal/core": "^8.8 || ^9.0",
+    "drupal/components": "^1.0 || ^2.0",
```

Comment:
Should we force the 2.0 upgrade and punch a new version?

Reply:
The 2.0 is for the new version of twig that comes with drupal 9. So we can't force that upgrade at this time.